lib/glific/flows.ex

```diff
@@ -680,6 +680,7 @@ defmodule Glific.Flows do
         fn flow, acc -> add_flow_keyword_map(flow, acc) end
       )
       |> add_default_flows(organization.out_of_office)
+      |> Map.put("new_contact", organization.newcontact_flow_id)
```

Comment:
should we call this: glific_default_new_contact  since new contact is a fairly common phrase, I dont want a potential NGO to use new_contact and conflict with our internal mapping